iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergingView.java

```diff
@@ -761,12 +681,8 @@ public int getDanglingLineCount() {
 
     @Override
     public DanglingLine getDanglingLine(final String id) {
-        return index.getNetworkStream()
-                .map(n -> n.getDanglingLine(id))
-                .filter(Objects::nonNull)
-                .map(index::getDanglingLine)
-                .findFirst()
-                .orElse(null);
+        final DanglingLine dl = index.get(n -> n.getDanglingLine(id), index::getDanglingLine);
+        return index.asDanglingLine(dl) ? dl : null;
```

Comment:
I'm not sure to understand the name of this method: is it a way to check if the danglingLine is merged or not?

Reply:
yes